packages/shared/src/utils/date/date-time.ts

```diff
@@ -1,55 +1,60 @@
 import moment from 'moment';
 import 'moment/min/locales';
 
+type TExtendedMoment = typeof moment & {
+    createFromInputFallback: (config: { _d: Date }) => void;
+};
+
 // Disables moment's fallback to native Date object
 // moment will return `Invalid Date` if date cannot be parsed
-moment.createFromInputFallback = function (config) {
+(moment as TExtendedMoment).createFromInputFallback = function (config: { _d: Date }) {
```

Comment:
```suggestion
(moment as TExtendedMoment).createFromInputFallback = function (config) {
```
@niloo-fs the type of `_d` is not necessary to declare any more because we have it in TExtendedMoment now🙂